src/results-main.js

```diff
@@ -238,7 +250,13 @@
   };
 
   var pollCallback = function(resp) {
-    var election = resp.AllElections[0];
+	const {
+		AllElections = []
+	} = resp;
+	const {
+		contests = []
+	} = AllElections.length > 0 ? AllElections[0] : [];
```

Comment:
since you repeat `AllElections[0]` on line 259 as well you could store this in a variable above, but not blocking.

Reply:
I like the idea of pulling the repeated variables into a function - just updated with 2 functions to safely pull these